src/main/java/ca/corefacility/bioinformatics/irida/service/remote/ProjectSynchronizationService.java

```diff
@@ -371,8 +374,16 @@ private void syncProject(Project project) {
 			}
 		}
 
-		//list the remote assemblies for the sample
-		List<UploadedAssembly> genomeAssembliesForSample = assemblyRemoteService.getGenomeAssembliesForSample(sample);
+		//list the remote assemblies for the sample.
+		List<UploadedAssembly> genomeAssembliesForSample;
+		try {
+			genomeAssembliesForSample = assemblyRemoteService.getGenomeAssembliesForSample(sample);
+		} catch (LinkNotFoundException e) {
+			//if the target IRIDA doesn't support assemblies yet, warn and ignore assemblies.
+			logger.warn("The referenced IRIDA instance doesn't support assemblies.");
```

Comment:
Would it be possible to print the URL to the referenced IRIDA instance?

Reply:
Sure.  Added in ce0cf1f